packages/logs/src/domain/assembly.spec.ts

```diff
@@ -12,11 +12,12 @@ import type { Clock } from '@datadog/browser-core/test'
 import { mockClock, cleanupSyntheticsWorkerValues, mockSyntheticsWorkerValues } from '@datadog/browser-core/test'
 import type { LogsEvent } from '../logsEvent.types'
 import type { CommonContext } from '../rawLogsEvent.types'
-import { getRUMInternalContext, resetRUMInternalContext, startLogsAssembly } from './assembly'
+import { startLogsAssembly } from './assembly'
 import { validateAndBuildLogsConfiguration } from './configuration'
 import { Logger, StatusType } from './logger'
 import type { LogsSessionManager } from './logsSessionManager'
 import { LifeCycle, LifeCycleEventType } from './lifeCycle'
+import { resetRUMInternalContext, getRUMInternalContext } from './rumInternalContext'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏move tests related to "internal context"  to `rumInternalContext.spec.ts`